examples/evs-csi-plugin/kubernetes/snapshot/sc.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: evs-sc
+provisioner: evs.csi.huaweicloud.com
+parameters:
+  type: SSD
+reclaimPolicy: Delete
```

Comment:
pls add a black line

Reply:
done, has changed in branch "feat-documents-yaml"